packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -1135,129 +1139,93 @@ import {
   DataListCheck,
   DataListControl,
   DataListDragButton,
-  DataListItemCells
+  DataListItemCells,
+  DragDrop,
+  Draggable,
+  Droppable
 } from '@patternfly/react-core';
 
+const getItems = count =>
+  Array.from({ length: count }, (v, k) => k).map(k => ({
+    id: `item-${k}`,
+    content: `item ${k} `.repeat(k === 4 ? 20 : 1)
+  }));
+
+const reorder = (list, startIndex, endIndex) => {
+  const result = Array.from(list);
+  const [removed] = result.splice(startIndex, 1);
+  result.splice(endIndex, 0, removed);
+  return result;
+};
+
 DraggableDataList = () => {
-  const [ liveText, setLiveText ] = React.useState('');
-  const [ id, setId ] = React.useState('');
-  const [ itemOrder, setItemOrder ] = React.useState(['data1', 'data2', 'data3', 'data4']);
+  const [items, setItems] = React.useState(getItems(10));
+  const [liveText, setLiveText] = React.useState('');
 
-  const onDragStart = newId => {
-    setId(newId)
-    setLiveText(`Dragging tarted for item id: ${newId}.`);
-  };
+  function onDrag(source) {
+    setLiveText(`Started dragging ${items[source.index].content}`);
+    return true;
```

Comment:
I think it'd be helpful to add a comment about why you are returning 'true' even though the DragDrop docs explain it.

Reply:
wait, did you do this? I don't see it.

Reply:
OH I just now see what you mean. I added it in DragDrop.md but not DataList.md, I'll add it now.